data/datacatalogue/CatalogueOntologies/Organisation roles.csv

```diff
@@ -1,7 +1,7 @@
-name
-Data originator
-Data provider
-Data holder
-Research
-Surveillance
-Other
\ No newline at end of file
+order,name,code,parent,ontologyTermURI,definition
+,Data originator,,,,
+,Data holder,,,https://spor.ema.europa.eu/rmswi/#/lists/200000029954/terms/200000029942,
+,Data provider,,,https://spor.ema.europa.eu/rmswi/#/lists/200000029954/terms/200000029943,
+,Research,,,https://spor.ema.europa.eu/rmswi/#/lists/200000029954/terms/200000029944,
```

Comment:
Following the ontologyTermURI the name should be _Researcher_ instead of Research